Vungle/src/main/java/com/mopub/mobileads/VungleRouter.java

```diff
@@ -160,15 +161,15 @@ void loadAdForPlacement(String placementId, VungleRouterListener routerListener)
             case INITIALIZED:
                 if (isValidPlacement(placementId)) {
                     addRouterListener(placementId, routerListener);
-                    Vungle.loadAd(placementId, loadAdCallback);
+                    Vungle.loadAd(placementId, adMarkup, adConfig, loadAdCallback);
```

Comment:
I am not sure should we keep this adMarkup in the map if adMarkup not null and our SDK not initialized? So that we can load it when SDK initialized.